IdentityCore/src/network/request_server_telemetry/MSIDAADTokenRequestServerTelemetry.m

```diff
@@ -47,15 +47,11 @@ - (instancetype)init
 - (void)handleError:(NSError *)error
             context:(id<MSIDRequestContext>)context
 {
-    if (!error) return;
-    
     NSString *errorString = [error msidServerTelemetryErrorString];
-    if (errorString)
-    {
-        [self.lastRequestTelemetry updateWithApiId:self.currentRequestTelemetry.apiId
-                                       errorString:errorString
-                                           context:context];
-    }
+    
+    [self.lastRequestTelemetry updateWithApiId:self.currentRequestTelemetry.apiId
+                                   errorString:errorString
+                                       context:context];
```

Comment:
Just for my understanding,  when reset if error is null,  ```self.silentSuccessfulCount``` is  reset to 0 instead of 1. Is there a place where it is added by 1 later on as it is the first success?

Reply:
We increase by calling `increaseSilentSuccessfulCount` when we read tokens from the cache.